packages/card/CardThumbnail.jsx

```diff
@@ -43,19 +36,31 @@ export default function CardThumbnail({
   return videoUrl ? (
     <>
       <button
-        className="su-component-card-thumbnail su-block su-relative su-z-10 su-size-full"
+        className="su-component-card-thumbnail su-group su-block su-relative su-z-10 su-size-full"
         type="button"
         aria-haspopup="dialog"
         onClick={() => handleClick()}
       >
         <MediaRatio
           imageUrl={imageUrl}
-          imageAlt={`${alt} (opens in a modal)`}
+          imageAlt={`Open video ${alt || ""} in a modal`}
```

Comment:
Small update to the `imageAlt` of the video preview. This is read out by the screen reader as the button text so I added "video" for clarity.